packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -127,7 +127,9 @@ abstract class SerializationManager {
   }
 
   /// Get the className for the provided object.
-  String? getClassNameForObject(Object data) {
+  String? getClassNameForObject(Object? data) {
+    if (data == null) return 'null';
+
     if (data is int) {
```

Comment:
Suggestion: Connect this with the other `if` `else if` statements bellow.

There is no reason to have this case in a different pattern.